README.md

```diff
@@ -86,8 +86,7 @@ JSON output based on a schema written in JSON. See [EDI](https://github.com/jf-t
 - [EdiFabric/EDI-Translator-Demo](https://github.com/EdiFabric/EDI-Translator-Demo) - EDI Translator for EDIFACT D.96A, X12 004010 and HIPAA 5010
 
 ## Public EDI References
-- [EDI Reference](https://edi.stedi.com/) - Free online viewer for all releases of X12 specifications
-- [EDI Mapping Guides](https://edi.stedi.com/mapping-guides) - Mapping Guides from Fortune 500 companies with free programmatic EDI validation to their guide.
```

Comment:
I am thinking we may be able to shortcut people into https://www.stedi.com/edi/x12 here. This page shows all the transaction sets all in one go.